Nautilus/Options/Attributes/ConfigFileMetadata.cs

```diff
@@ -56,26 +56,24 @@ public void ProcessMetadata()
 
     private void processMetadata()
     {
-        BindingFlags bindingFlags = BindingFlags.Instance | BindingFlags.Public;
-        foreach (PropertyInfo property in typeof(T).GetProperties(bindingFlags)
-                     .Where(memberIsDeclaredInConfigFileSubclass) // Only care about members declared in a subclass of ConfigFile
-                     .Where(memberIsNotIgnored)) // Filter out explicitly ignored members
+        BindingFlags bindingFlags = BindingFlags.Instance | BindingFlags.Public | BindingFlags.Static;
+        
+        foreach (MemberInfo member in typeof(T).GetMembers(bindingFlags)
+                     .Where(memberIsDeclaredInConfigFileSubclass)
+                     .Where(memberIsNotIgnored))
         {
-            processFieldOrProperty(property, MemberType.Property, property.PropertyType);
-        }
-
-        foreach (FieldInfo field in typeof(T).GetFields(bindingFlags)
-                     .Where(memberIsDeclaredInConfigFileSubclass) // Only care about members declared in a subclass of ConfigFile
-                     .Where(memberIsNotIgnored)) // Filter out explicitly ignored members
-        {
-            processFieldOrProperty(field, MemberType.Field, field.FieldType);
-        }
-
-        foreach (MethodInfo method in typeof(T).GetMethods(bindingFlags | BindingFlags.Static)
-                     .Where(memberIsDeclaredInConfigFileSubclass) // Only care about members declared in a subclass of ConfigFile
-                     .Where(memberIsNotIgnored)) // Filter out explicitly ignored members
-        {
-            processMethod(method);
+            switch (member)
+            {
+                case FieldInfo field:
+                    processFieldOrProperty(field, MemberType.Field, field.FieldType);
```

Comment:
The previous code checked that fields were not static.

```suggestion
                    if (!field.IsStatic)
                        processFieldOrProperty(field, MemberType.Field, field.FieldType);
```